### DISCUSSION TITLE
tmux-powerline

### DISCUSSION THREAD

**Discussion by srdtrk** [catppuccin/catppuccin#2366] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

There is already an [example theme](https://github.com/erikw/tmux-powerline/blob/main/themes/bubble.sh) based on catppuccin's macchiato flavour.

### Any additional comments?

The example theme is only partially complete since only the uncommented elements are themed. If the user wants the enable other fields, they'd need to override the default colors. I could change this during the port.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Closing as further discussion should take place in #2559
